multipy/runtime/interpreter/plugin_torch.cpp

```diff
@@ -0,0 +1,48 @@
+#include <torch/csrc/jit/python/pybind_utils.h>
+#include <torch/csrc/lazy/core/debug_util.h>
+
+#include "plugin_registry.h"
+
+namespace multipy {
+namespace torch {
+
+namespace {
+std::vector<::torch::lazy::SourceLocation> NoPythonFrames() {
+  ::torch::lazy::SourceLocation dummy_loc;
+  dummy_loc.file = "No Python Frames";
+  return {dummy_loc};
+}
+} // namespace
+
+class TorchConverter : public Converter {
+ public:
+  TorchConverter() {
```

Comment:
Might be a naive question, but where does this get called / when do TorchConverters get created to make this?

Reply:
if you see below this is statically initialized and registered at load time -- only created once per interpreter